### PR TITLE
Unite testing

### PR DESCRIPTION
**Pull request by rustam-tolipov**
### OOP school library: unit tests

We created tests for these classes:

- `Book` 💯 
- `Person` 💯 
- `Student` 💯 
- `Teacher` 💯 
- `Classroom` 💯 
- `Rentals` 💯 

Feel free to review🧑🏻‍💻  our test and let us know if there are any changes to do. 🤓. 🤓.

### CODE REVIEW

File: spec/person_spec.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      expect(person.age).to eq(20)
+      expect(person.parent_permission).to eq({ parent_permission: true })
+    end
+  end
+  describe '#rentals' do
+    it 'should return an empty array' do
+      person = Person.new('Rustamjon', 20)
+      expect(person.rentals).to eq([])
+    end
+  end
```
Comment (Dipeshtwis) [inline spec/person_spec.rb]: - [x] I would recommend adding more tests for `adding a new rental` by making instances of `book` and `person` and assigning it to rentals.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @Rustamxon7 and @ahzia, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: All tests are passing. :heavy_check_mark: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comment under the review_**

- [ ] README file should have a command on how to run the test. [Hint: `rspec`]

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/person_spec.rb (hunk 1 above)
Comment (rustam-tolipov) [reply spec/person_spec.rb]: Thanks for reminding this issue!

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi Team, 

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**